### PR TITLE
add disable auto failover

### LINKED ISSUE #492: Improve failover doc
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

Enable Automatic Failover -> Configure Automatic Failover

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### LINKED ISSUE #790: Add procedure about how to disable auto-failover function of TiDB Operator
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by BinChenn**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

fix #492, change catalog name

fix #790, add how to disable auto failover and failover notes

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/use-auto-failover.md [hunk 1]
```diff
@@ -10,7 +10,17 @@ Automatic failover means that when a node in the TiDB cluster fails, TiDB Operat
...
-The automatic failover feature is enabled by default in TiDB Operator. You can disable it by setting `controllerManager.autoFailover` to `false` in the `charts/tidb-operator/values.yaml` file when deploying TiDB Operator:
+## Configure Automatic Failover
+
+The automatic failover feature is enabled by default in TiDB Operator. 
+
+> **Note:**
+>
+> If there are not enough resources in the cluster for TiDB Operator to create new nodes, the automatic failover feature will not take effect.
+
+## Disable Automatic Failover
```
Comment (ran-huang) [inline en/use-auto-failover.md:21]:
```suggestion
### Disable automatic failover
```

File: en/use-auto-failover.md [hunk 2]
```diff
@@ -10,7 +10,17 @@ Automatic failover means that when a node in the TiDB cluster fails, TiDB Operat
 
 TiDB Operator manages Pods based on `StatefulSet`, which does not automatically create a new node to replace the original node when a Pod goes down. For this reason, the automatic failover feature is added to TiDB Operator, which expands the behavior of `StatefulSet`.
 
-The automatic failover feature is enabled by default in TiDB Operator. You can disable it by setting `controllerManager.autoFailover` to `false` in the `charts/tidb-operator/values.yaml` file when deploying TiDB Operator:
+## Configure Automatic Failover
```
Comment (ran-huang) [inline en/use-auto-failover.md:13]:
```suggestion
## Configure automatic failover
```

File: en/use-auto-failover.md [hunk 3]
```diff
@@ -10,7 +10,17 @@ Automatic failover means that when a node in the TiDB cluster fails, TiDB Operat
...
+
+The automatic failover feature is enabled by default in TiDB Operator. 
+
+> **Note:**
+>
+> If there are not enough resources in the cluster for TiDB Operator to create new nodes, the automatic failover feature will not take effect.
+
+## Disable Automatic Failover
+
+You can disable it by setting `controllerManager.autoFailover` to `false` in the `charts/tidb-operator/values.yaml` file when deploying TiDB Operator, example is as follows:
```
Comment (ran-huang) [inline en/use-auto-failover.md:23]:
```suggestion
You can disable it by setting `controllerManager.autoFailover` to `false` in the `charts/tidb-operator/values.yaml` file when deploying TiDB Operator. For example:
```

File: zh/use-auto-failover.md [hunk 4]
```diff
@@ -10,7 +10,17 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/use-auto-failover/']
 
 由于 TiDB Operator 基于 `StatefulSet` 来管理 Pod，但 `StatefulSet` 在某些 Pod 发生故障时不会自动创建新节点来替换旧节点，所以，TiDB Operator 扩展了 `StatefulSet` 的这种行为，添加了 Auto Failover 功能。
 
-Auto Failover 功能在 TiDB Operator 中默认开启。部署 TiDB Operator 时，可通过设置 `charts/tidb-operator/values.yaml` 文件的 `controllerManager.autoFailover` 为 `false` 关闭该功能：
+## 配置故障自动转移
+
+Auto Failover 功能在 TiDB Operator 中默认开启。
```
Comment (ran-huang) [inline zh/use-auto-failover.md:15]:
```suggestion
故障自动转移功能在 TiDB Operator 中默认开启。
```

File: zh/use-auto-failover.md [hunk 5]
```diff
@@ -10,7 +10,17 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/use-auto-failover/']
...
+
+> **注意：**
+> 
+> 如果集群中没有足够的资源以供 TiDB Operator 创建新节点，则 Auto Failover 功能不会生效。
```
Comment (ran-huang) [inline zh/use-auto-failover.md:19]:
```suggestion
> 如果集群中没有足够的资源以供 TiDB Operator 创建新节点，则故障自动转移功能不会生效。
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by handlerww — APPROVED** [review]

File: en/use-auto-failover.md (hunk 1 above)
Comment (BinChenn) [reply en/use-auto-failover.md]: fixed

File: en/use-auto-failover.md (hunk 2 above)
Comment (BinChenn) [reply en/use-auto-failover.md]: fixed

File: en/use-auto-failover.md (hunk 3 above)
Comment (BinChenn) [reply en/use-auto-failover.md]: fixed

File: zh/use-auto-failover.md (hunk 4 above)
Comment (BinChenn) [reply zh/use-auto-failover.md]: fixed

File: zh/use-auto-failover.md (hunk 5 above)
Comment (BinChenn) [reply zh/use-auto-failover.md]: fixed

**Review by ran-huang — APPROVED** [review]

**Comment by DanielZhangQD** [comment]
@BinChenn @handlerww @ran-huang The description of `If there are not enough resources in the cluster for TiDB Operator to create new nodes, the automatic failover feature will not take effect.` is incorrect, the failover will still take effect and scale out new Pods, but the Pods will be in Pending state and it will not make up the failed Pods.

**Comment by ran-huang** [comment]
> @BinChenn @handlerww @ran-huang The description of `If there are not enough resources in the cluster for TiDB Operator to create new nodes, the automatic failover feature will not take effect.` is incorrect, the failover will still take effect and scale out new Pods, but the Pods will be in Pending state and it will not make up the failed Pods.

@BinChenn Could you fix this description?

**Comment by BinChenn** [comment]
> > @BinChenn @handlerww @ran-huang The description of `If there are not enough resources in the cluster for TiDB Operator to create new nodes, the automatic failover feature will not take effect.` is incorrect, the failover will still take effect and scale out new Pods, but the Pods will be in Pending state and it will not make up the failed Pods.
> 
> @BinChenn Could you fix this description?

I will fix this error in another PR.

**Pull request merged**